cypress/features/regression/experimental/textboxMultiple.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Experimental Textbox multiple component
+  I want to change Experimental Textbox multiple component properties
+
+  Background: Open Experimental Textbox multiple component page
+    Given I open "Experimental Textbox" component page multiple
+ 
+  @positive
+  Scenario Outline: Set placeholder to <placeholder>
+    When I set placeholder to "<placeholder>"
+    Then "First" textbox placeholder is set to "<placeholder>"
```

Comment:
the same is here, if we have 1 parameter that changes something in both inputs, we could put them into one stem and avoid `First` & `Second` just replace them by Multiple textbox

Reply:
fixed